docs/reference/ccr/index.asciidoc

```diff
@@ -2,28 +2,300 @@
+
+[discrete]
+[[ccr-centralized-reporting]]
+==== Centralized reporting
+Using a centralized reporting cluster is useful when querying across a large
+network is inefficient. In this configuration, you replicate data from many
+smaller clusters to the centralized reporting cluster.
+
+For example, a large global bank might have 100 {es} clusters around the world
+that are distributed across different regions for each bank branch. Using
+{ccr}, the bank can replicate events from all 100 banks to a central cluster to
+analyze and aggregate events locally for reporting. Rather than maintaining a
+mirrored cluster, the bank can use {ccr} to replicate specific indices.
+
+In the following diagram, data from three datacenters in different regions is
+replicated to a centralized reporting cluster. This configuration enables you
+to copy data from regional hubs to a central cluster, where you can run all
+reports locally.
+
+image::images/ccr-arch-central-reporting.png[Three clusters in different regions sending data to a centralized reporting cluster for analysis]
+
+[discrete]
+[[ccr-replication-mechanics]]
+=== Replication mechanics
+Although you <<ccr-getting-started,set up {ccr}>> at the index level, {es}
+achieves replication at the shard level. When a follower index is created,
+each shard in that index pulls changes form its corresponding shard in the
+leader index, which means that a follower index has the same number of
+shards as its leader index. All operations are replicated by the follower, so that operations to create, update, or delete a document are replicated.
+These requests can be served from any copy of the leader shard (primary or
+replica).
+
+When a follower shard sends a read request, the leader shard responds with
+any new operations, limited by the read parameters that you establish when
+configuring the follower index. If no new operations are available, the
+leader shard waits up to the configured timeout for new operations. If the
+timeout elapses, the leader shard responds to the follower shard that there
+are no new operations. The follower shard updates shard statistics and
+immediately sends another read request to the leader shard. This
+communication model ensures that network connections between the remote
+cluster and the local cluster are continually in use, avoiding forceful
+termination by an external source such as a firewall.
+
+If a read request fails, the cause of the failure is inspected. If the
+cause of the failure is deemed to be recoverable (such as a network
+failure), the follower shard enters into a retry loop. Otherwise, the
+follower shard pauses
+<<ccr-pause-replication,until you resume it>>.
+
+When a follower shard receives operations from the leader shard, it places
+those operations in a write buffer. The follower shard uses the write buffer
+to submit bulk write requests within the shard. If the write buffer exceeds
```

Comment:
> The follower shard uses the write buffer to submit bulk write requests within the shard

I think we need to adjust this sentence. The follower shard submits bulk write requests using operations from the write buffer.

Reply:
Changed to read:
>The follower shard submits bulk write requests using operations from the write buffer.